source/loader/layers/sanitizer/linux/backtrace.cpp

```diff
@@ -15,22 +15,22 @@
 namespace ur_sanitizer_layer {
 
 StackTrace GetCurrentBacktrace() {
-    void *Frames[MAX_BACKTRACE_FRAMES];
+    BacktraceFrame Frames[MAX_BACKTRACE_FRAMES];
     int FrameCount = backtrace(Frames, MAX_BACKTRACE_FRAMES);
-    char **Symbols = backtrace_symbols(Frames, FrameCount);
-
-    if (Symbols == nullptr) {
-        return StackTrace();
-    }
 
     StackTrace Stack;
-    for (int i = 0; i < FrameCount; i++) {
-        BacktraceInfo addr_info(Symbols[i]);
-        Stack.stack.emplace_back(addr_info);
-    }
-    free(Symbols);
+    Stack.stack =
```

Comment:
nit, we should be able to skip the stack buffer by writing to the vector directly:
```
StackTrace Stack;
Stack.stack.reserve(MAX_BACKTRACE_FRAMES);
int FrameCount = backtrace(Stack.stack.data(), MAX_BACKTRACE_FRAMES);
```

Reply:
I didn't want to delay merging because of this. If you want to change it, include it in another PR.

Reply:
Thanks. I'll do this later.